modules/solid_mechanics/include/materials/ReturnMappingModel.h

```diff
@@ -57,7 +57,6 @@ class ReturnMappingModel : public ConstitutiveModel
   Real _effective_strain_increment;
 
 private:
```

Comment:
Looks like you address all of Daniel's comments except this one. I'm not sure if this is a showstopper but could be cleaned up. @dschwen, any further comments?